spec/rubocop/cop/rspec/empty_example_group_spec.rb

```diff
@@ -19,6 +19,20 @@
         it { should be_true }
       end
     RUBY
+
+    expect_correction(<<~RUBY)
+      describe Foo do
+      #{'  '}
```

Comment:
Is this the leftover indentation whitespace?
Would `range_by_whole_lines` help removing the empty group along with its lines, and potentially with a following/leading separator newline?

Reply:
Oh, that's a nice feature! I used it right away and it looks very good.
I added https://github.com/rubocop/rubocop-rspec/commit/bd0f3bce51cbe65b2d93da7fb23a55fa64033e84 (then rebased 2 commits & force-pushed).